src/library_syscall.js

```diff
@@ -1118,7 +1118,10 @@ var SyscallsLibrary = {
         return 0;
       }
       case {{{ cDefine('F_GETLK') }}}:
-      case {{{ cDefine('F_GETLK64') }}}: {
+      /* case {{{ cDefine('F_GETLK64') }}}: Currently in musl F_GETLK64 has same value as F_GETLK, but if that changes, uncomment this */ {
+#if ASSERTIONS
+        assert({{{ cDefine('F_GETLK') }}} === {{{ cDefine('F_GETLK64') }}});
```

Comment:
let's make this a static assert, that is, that runs at compile time, and does not depend on this code being reached at runtime. we can do it like this:
```
{{{ assert(cDefine('F_GETLK') === cDefine('F_GETLK64')), '' }}}
```
(we need to return an empty string, hence the `''`)

Reply:
(same for the other assert later down)

Reply:
ok